src/NzbDrone.Core/Indexers/IndexerFactory.cs

```diff
@@ -15,6 +15,8 @@ public interface IIndexerFactory : IProviderFactory<IIndexer, IndexerDefinition>
 
     public class IndexerFactory : ProviderFactory<IIndexer, IndexerDefinition>, IIndexerFactory
     {
+        private readonly IIndexerRepository _providerRepository;
```

Comment:
We'd have to have the RemoveImplementations in the ProviderFactory, in this case its actually sufficient to remove it and `RemoveMissingImplementations()` will take care of it for us, but thats only because Animezb has a NullConfig, if it had a its own Config this method would fail since it couldn't deserialize from the DB. I'll clean this up and make a note to fix it up.